tap-snapshots/test-parse-test.js-TAP.test.js

```diff
@@ -2195,7 +2195,7 @@ Object {
     },
     "message": "Assignment to constant variable foo (2:2)",
     "pos": 4,
-    "type": "TypeError",
+    "type": "SyntaxError",
```

Comment:
You can run TAP_SNAPSHOT=1 to regenerate snapshots.

Reply:
I made this change through GitHub.